frontend/src/app/libs/common-components/components/slide-toggle/slide-toggle.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit, Input, Output, ChangeDetectionStrategy, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-slide-toggle',
+  templateUrl: './slide-toggle.component.html',
+  styleUrls: ['./slide-toggle.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SlideToggleComponent {
+
+  @Input()
+  public isDisabled: boolean = true;
+
+  @Input()
+  public isDarkTheme: boolean = false;
+
+  @Output()
+  public onToggle: EventEmitter<any> = new EventEmitter();
+
+	public isActive: boolean = false;
+
+  public toggle(): void {
```

Comment:
i see an output for value, but not seeing any `@Input() value: boolean` for value